src/common/plugin/makeCurrencyPlugin.ts

```diff
@@ -21,22 +22,13 @@ export function makeCurrencyPlugin(
   const { io, log, pluginDisklet } = pluginOptions
   const currencyTools = makeCurrencyTools(io, pluginInfo)
   const { defaultSettings, pluginId, currencyCode } = currencyInfo
-  const {
-    customFeeSettings,
-    serverList,
-    disableFetchingServers
-  } = defaultSettings
   const state = makePluginState({
     io,
     currencyCode,
     pluginId,
     pluginDisklet,
     log,
-    defaultSettings: {
-      customFeeSettings,
-      serverList,
-      disableFetchingServers
-    }
+    defaultSettings: asUtxoUserSettings(defaultSettings)
```

Comment:
Why are we cleaning the `defaultSettings` here if it's coming from `currencyInfo` which should have a trusted type?